app/models/user.rb

```diff
@@ -8,6 +8,7 @@
 #  email      :string(255)      not null
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
+#  admin      :boolean          default(FALSE)
 #
 class User < ApplicationRecord
```

Comment:
Could add

`validates :admin, presence: true`

And in spec:
 `it { should validate_presence_of(:admin) }`